content/tutorials/manage-the-rippled-server/configuration/configure-full-history.md

```diff
@@ -0,0 +1,92 @@
+
+1. Stop the `rippled` server if it is running.
+
+        $ sudo systemctl stop rippled
+
+0. Remove (or comment out) the `online_delete` and `advisory_delete` settings from the `[node_db]` stanza of your server's config file, and change the type to `NuDB` if you haven't already:
+
+        [node_db]
+      	type=NuDB
+      	path=/var/lib/rippled/db/nudb
+      	#online_delete=2000
+      	#advisory_delete=0
+
+    On a full-history server, you should use NuDB for the ledger store, because RocksDB requires too much RAM when the database is that large. For more information, see [Capacity Planning](capacity-planning.html). You can remove the following performance-related configuration options from the default `[node_db]` stanza, because they only apply to RocksDB: `open_files`, `filter_bits`, `cache_mb`, `file_size_mb`, and `file_size_mult.`
+
+    **Caution:** If you have any history already downloaded with RocksDB, you must either delete that data or change the paths to the databases in the config file when you switch to NuDB. You must change both the `path` field of the `[node_db]` stanza **and** the `[database_path]` (SQLite database) setting. Otherwise, the server may [fail to start](server-wont-start.html#state-db-error).
+
+    {% include '_snippets/conf-file-location.md' %}<!--_ -->
+
+0. Set the `[ledger_history]` stanza of your server's config file to `full`:
+
+        [ledger_history]
+        full
+
+0. Set the `[ips_fixed]` stanza of your server's config file to explicitly peer with at least one server that has full history available.
+
+        [ips_fixed]
+        169.55.164.20
+        50.22.123.215
+
+    Your server can only download historical data from the peer-to-peer network if one its direct peers has the data available. The easiest way to ensure you can download full history is to peer with a server that already has full history.
+
+    **Tip:** Ripple makes a pool of full history servers publicly available. You can resolve the domain `s2.ripple.com` a few times to get the IP addresses of these servers. Ripple offers these servers as a public service, so be aware that their availability to peer with other servers is limited and you may be blocked if you abuse them.
+
+0. If you have a database dump from another full-history server to use as a basis, set the `[import_db]` stanza of your server's config file to point to the data to be imported. (Otherwise, skip this step.)
+
+        [import_db]
+      	type=NuDB
+      	path=/tmp/full_history_dump/
+
+0. Remove your server's existing database files, if you have any from previously running `rippled`.
+
+    After disabling online deletion, the server ignores any data that was downloaded while online deletion was enabled, so you may as well clear up the disk space. For example:
+
+        rm -r /var/lib/rippled/db/rocksdb/*
+        rm /var/lib/rippled/db/*.db
+        rm /var/lib/rippled/db/*.sqlite
+
```

Comment:
I've seen SQLite sometimes createing files with a slightly different extension. Basically, wipe out the whole directory:
rm -fr /var/lib/rippled/db/*